contrib/interconnect/ic_modules.c

```diff
@@ -137,29 +137,47 @@ MotionIPCLayer udpifc_ipc_layer = {
     .GetMotionConnTupleRemapper = GetMotionConnTupleRemapper,
 };
 
+MotionIPCLayer *ipc_layer_impls[] = {
+	&tcp_ipc_layer,
+	&udpifc_ipc_layer,
+	&proxy_ipc_layer,
+};
+
 void
 _PG_init(void)
 {
+	int elevel, impls_num;
```

Comment:
It's not a common style to define two variables in one line.

Reply:
fix with [commit](https://github.com/cloudberrydb/cloudberrydb/pull/646/commits/2a8be3e5d0d894c98a0f4854c1c07138190cfd21)